presto-kudu/src/test/java/io/prestosql/plugin/kudu/TestKuduIntegrationDecimalColumns.java

```diff
@@ -80,6 +80,23 @@ public void testDecimalColumn()
         assertUpdate("DROP TABLE test_decimal");
     }
 
+    @Test
+    public void testDeleteByPrimaryKeyDecimalColumn()
+    {
+        assertUpdate("CREATE TABLE IF NOT EXISTS test_decimal (" +
+            "decimal_id decimal(18, 3) WITH (primary_key=true), " +
+            "col_decimal decimal(18, 3)" +
+            ") WITH (" +
+            " partition_by_hash_columns = ARRAY['id'], " +
+            " partition_by_hash_buckets = 2" +
+            ")");
+
+        assertUpdate("INSERT INTO test_decimal VALUES (1.1, 1.1), (2.2, 2.2), (3.3, 3.3)", 3);
+        assertUpdate("DELETE FROM test_decimal WHERE decimal_id = 2.2", 1);
```

Comment:
You could assert here on SELECT to test whether a row was actually deleted.

Reply:
Yes, that's cool, thank you!
Actually I think when it return 1 in `assertUpdate` it means the delete action is succeed.    :)

Reply:
@wendigo Thank you for your suggestion, I have added the `select` sql.   :)